src/components/DetailsPanelQuery.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <v-details-panel-header object-type="SQL query" :title="title" />
+    <v-details-panel-list title="Events" :items="object.events" />
+  </div>
+</template>
+
+<script>
+import VDetailsPanelHeader from '@/components/DetailsPanelHeader.vue';
+import VDetailsPanelList from '@/components/DetailsPanelList.vue';
+import { getSqlLabelFromString } from '@/lib/util';
+
+export default {
+  name: 'v-details-panel-function',
```

Comment:
Can we check if `events` only contains a single event, and if it does, automatically select it?

Reply:
Done. https://github.com/applandinc/appmap-js/pull/132/commits/267ef09fa0ae3b9da25552747ff4230d4d11ac96